### PR TITLE
Fix magic numbers in Celebi event code

### PR DESCRIPTION
**Pull request by Idain**

### CODE REVIEW

File: engine/events/celebi.asm [hunk 1]
```diff
@@ -132,7 +132,7 @@ UpdateCelebiPosition:
 	ld hl, SPRITEANIMSTRUCT_YCOORD
 	add hl, bc
 	ld a, [hl]
-	cp 8 * 10 + 2
+	cp TILE_WIDTH * 10 + 2
```
Comment (Rangi42) [inline engine/events/celebi.asm:135]: Everywhere else uses <code><i>NUM</i> * TILE_WIDTH</code>.

File: engine/events/celebi.asm [hunk 2]
```diff
@@ -159,9 +159,9 @@ UpdateCelebiPosition:
 	ld hl, SPRITEANIMSTRUCT_XCOORD
 	add hl, bc
 	add [hl]
-	cp 8 * 11 + 4
+	cp TILE_WIDTH * 11 + 4
 	jr nc, .ShiftY
-	cp 8 *  8 + 4
+	cp TILE_WIDTH *  8 + 4
```
Comment (Rangi42) [inline engine/events/celebi.asm:164]: Good opportunity to *not* use that double-space alignment.

File: engine/events/celebi.asm (hunk 1 above)
Comment (Idain) [reply engine/events/celebi.asm]: Done.

File: engine/events/celebi.asm (hunk 2 above)
Comment (Idain) [reply engine/events/celebi.asm]: Aaand done.

**Review by Rangi42 — APPROVED** [review]

**Comment by mid-kid** [comment]
Merged as https://github.com/pret/pokecrystal/commit/bbfb4ab60e8e4450399fa8168c28cd71e837b3ea

**Pull request closed without merging**